pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java

```diff
@@ -418,6 +431,15 @@ public PulsarClient getReplicationClient(String cluster) {
     }
 
     private CompletableFuture<Topic> createPersistentTopic(final String topic) throws RuntimeException {
+        try {
+            return createPersistentTopic(topic, false);
```

Comment:
yes, we can add comment. But the idea of having this flag:
1. when `BrokerService` receives `createPersistentTopic` request, passing `failOnTooManyConcurrentTopics=false` means we will not fail the request due to throttling but will add into the queue.
2. But when we retry from `pendingTopicLoadingQueue` and again if it gets throttle then we add that topic-future again into the queue at [line1037](https://github.com/rdhabalia/pulsar/blob/406b0fc6b8efcc1451e163858a1e3f72860968fd/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java#L1037)